packages/pdfshift/src/Utils.js

```diff
@@ -0,0 +1,41 @@
+import { composeNextState } from '@openfn/language-common';
+import {
+  request as commonRequest,
+} from '@openfn/language-common/util';
+import nodepath from 'node:path';
+
+export const prepareNextState = (state, response) => {
+  const { body, ...responseWithoutBody } = response;
+
+  if (!state.references) {
+    state.references = [];
+  }
+
+  return {
+    ...composeNextState(state, response.body),
+    response: responseWithoutBody,
+  };
+};
+
+export const request = (configuration = {}, method, path, options) => {
+  const { baseUrl= 'https://api.pdfshift.io', apiKey, apiVersion = 'v3' } = configuration;
```

Comment:
Would baseURL ever change? Do we even need to expose this as config?

Reply:
hmm, but also, can you not just do this?
```
const testServer = enableMockClient('https://api.pdfshift.io');
```

Reply:
I removed it from config and my tests were reading the default baseURL from utils

Reply:
> Ah I see. In that case I would leave it on state.config but don't document it - make it a secret property that we can use in testing, but that doesn't confuse users

@josephjclark  How can I achieve this with the baseUrl?

Reply:
I get it. I have removed it. I was making a mistake in Utils which was making the tests fail, but its good now